src/autogluon/bench/Dockerfile

```diff
@@ -1,5 +1,27 @@
 ARG AG_BENCH_BASE_IMAGE
 FROM $AG_BENCH_BASE_IMAGE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install essential packages and Python 3.9
+RUN apt-get update && \
+    apt-get install -y software-properties-common build-essential && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get install -y python3.9 python3.9-dev python3.9-distutils python3.9-venv && \
```

Comment:
Doesn't need to be addressed in this PR, but maybe we can expose an ARG and let user to pass in the python version they want to run benchmark on